cli-rs/src/main.rs

```diff
@@ -359,9 +359,14 @@ fn compile_riscv_asm<F: FieldElement>(
         None => powdr_riscv::Runtime::base(),
     };
 
-    powdr_riscv::compile_riscv_asm::<F>(
+    powdr_riscv::compile_riscv_asm_bundle::<F>(
```

Comment:
is elf still the default compilation path here?

Reply:
There is no ELF path in this function, this only from assembly.